packages/components/message-box/src/index.vue

```diff
@@ -335,7 +335,11 @@ export default defineComponent({
       () => visible.value,
       (val) => {
         if (val) {
-          if (props.boxType === 'alert' || props.boxType === 'confirm') {
+          if (
+            props.boxType === '' ||
```

Comment:
boxType = '' | 'prompt' | 'alert' | 'confirm'

Maybe just exclude the 'prompt' here